getting-started/display.rst

```diff
@@ -49,6 +49,12 @@ Next, start up a Python interpreter.
 You can use the default Python shell (:command:`python`), the `IPython shell`_, or even run from a `Jupyter Notebook`_.
 Ensure that this Python session is running from the shell where you ran :command:`setup lsst_distrib`.
 
+If the image does not display with DS9, try these troubleshooting steps:
+
+ * launching DS9 *after* setting up the LSST environment in a new terminal window
+ * installing ``xpa-tools`` with ``sudo apt install xpa-tools; xpans``
```

Comment:
Is this specifically a section about Ubuntu? If not, this line would be confusing to users of any other system (e.g. `rpm` on RedHat derivatives, and there is no equivalent on macos). Instead, maybe just link to the prereqs page and say "you may need to install additional applications..." or something like that?

Reply:
Added at the start of the second bullet "if using `Debian/Ubuntu`_" to reference the prereqs page and make it clear that tip is only for Ubuntu